cmd/loadTests.go

```diff
@@ -190,6 +206,11 @@ func setup(cmd *cobra.Command, args []string) {
 	cmd.SilenceUsage = true
 	term := terminal.New(cmd.InOrStdin, cmd.OutOrStdout, verbose)
 
+	// waitDeployments is valid only if waitPipelines
+	if waitDeployments && !waitPipelines {
+		klog.Fatalf("Error: The --waitdeployments flag requires the --waitpipelines flag.")
+	}
+
```

Comment:
```suggestion
	// waitDeployments sets waitPipelines=true implicitly
	waitPipelines = waitPipelines || waitDeployments
```

Instead of requiring to set both flags every time `--waitdeployments` is set (which is unnecessarily verbose and tedious) what about setting the `--waitpipelines` to `true` implicitly if `--waitdeployments` is set?

Reply:
If you accept the suggestion, please update the docs and help accordingly.

Reply:
Updated